### PR TITLE
HCAP-421 Add link to BC homepage

### PR DESCRIPTION
**Pull request by arranfw**
Modify logo alt for better clarity

https://freshworks.atlassian.net/browse/HCAP-421

- [ ]  Tested on dev

### CODE REVIEW

**Review by Zenen-freshworks — APPROVED** [review]
Seems like a straightforward PR!

File: client/src/components/generic/Header.js [hunk 1]
```diff
@@ -96,22 +95,24 @@ export const Header = ({hideEmployers=false}) => {
...
+                  src={BcLogo}
+                  alt="Government of british columbia home"
+                  onClick={handleLogoClick}
+                />
+              </Hidden>
+              <Hidden mdUp>
+                <img
+                  className={classes.logo}
+                  src={BcLogoMini}
+                  alt="Government of british columbia home"
```
Comment (ashtonmeuser) [inline client/src/components/generic/Header.js:111]: Nitpick: casing.
```suggestion
                  alt="Government of British Columbia"
```

File: client/src/components/generic/Header.js [hunk 2]
```diff
@@ -96,22 +95,24 @@ export const Header = ({hideEmployers=false}) => {
...
-                alt="Government of B.C."
-                onClick={handleLogoClick}
-              />
-            </Hidden>
+            <a href="https://www2.gov.bc.ca/gov/content/home">
+              <Hidden smDown>
+                <img
+                  className={classes.logo}
+                  src={BcLogo}
+                  alt="Government of british columbia home"
```
Comment (ashtonmeuser) [inline client/src/components/generic/Header.js:103]:
```suggestion
                  alt="Government of British Columbia"
```

**Review by ashtonmeuser — CHANGES_REQUESTED** [review]

**Review by ashtonmeuser — DISMISSED** [review]

**Review by ashtonmeuser — APPROVED** [review]

File: client/src/components/generic/Header.js [hunk 3]
```diff
@@ -49,7 +49,6 @@ const useStyles = makeStyles((theme) => ({
   logo: {
     height: '40px',
     marginBottom: theme.spacing(1.1),
-    cursor: 'pointer',
     flexShrink: 0, // Logo should hold its own
```
Comment (Zenen-freshworks) [inline client/src/components/generic/Header.js:52]: this logo ain't no pushover

**Review by Zenen-freshworks — APPROVED** [review]

**Pull request merged**